frontends/web/src/app.tsx

```diff
@@ -94,7 +96,7 @@ export const App = () => {
       return;
     }
     // if on index page and have at least 1 account, route to /account-summary
-    if (isIndex && accounts.length) {
+    if (isIndex && !noAccounts) {
```

Comment:
Also the next if conditions can exploit `noAccounts`

Reply:
I mean
```
    if (accounts.length === 0 && currentURL.startsWith('/buy/')) {
      navigate('/');
      return;
    }
    // if on the /bitsurance/ view and there are no accounts view route to /
    if (accounts.length === 0 && currentURL.startsWith('/bitsurance/')) {
      navigate('/');
      return;
    }
```

Reply:
I didn't add it there as these conditions (bitsurance and buy) are related to mainnet accounts.